src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,13 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
```

Comment:
lets test your code from main, create a few drivers, get them and see what happens

Reply:
I tested my code in the main method however i deleted it